float32[] pose_obj
